app/grants/models/contribution.py

```diff
@@ -18,6 +18,7 @@ class Contribution(SuperModel):
     CHECKOUT_TYPES = [
         ('eth_std', 'eth_std'),
         ('eth_zksync', 'eth_zksync'),
+        ('eth_polygon', 'eth_polygon'),
```

Comment:
Oh this is def need piece ! @chibie

Reply:
oh shit! all my polygon backend changes in former models.py (https://github.com/gitcoinco/web/blob/5901de79753445f0c76cd406f49154186e0e80d7/app/grants/models.py) are missing from contribution.py in master!!  😕

Reply:
@chibie Oh no! I thought I had updated it with all the changes 😞